publication-commons/src/main/java/no/unit/nva/publication/model/business/TicketEntry.java

```diff
@@ -34,14 +35,16 @@ static <T extends TicketEntry> TicketEntry createNewTicket(Publication publicati
     }
     
     static Stream<Class<? extends TicketEntry>> ticketTypes() {
-        return Stream.of(DoiRequest.class, PublishingRequestCase.class);
+        return Stream.of(DoiRequest.class, PublishingRequestCase.class, GeneralSupportRequest.class);
```

Comment:
We do not use the the jackson subtypes to generate the stream because Message is still a ticket Entry